tests/Operation/FindFunctionalTest.php

```diff
@@ -36,11 +38,16 @@ function (array $event) use ($expectedQuery): void {
         );
     }
 
-    /** @dataProvider provideModifierDocuments */
+    #[DataProvider('provideModifierDocuments')]
     public function testModifierDocuments($modifiers, stdClass $expectedSort): void
     {
         (new CommandObserver())->observe(
             function () use ($modifiers): void {
+                // @todo revert this lines after PHPC-2457
```

Comment:
Technically, this should become irrelevant after https://github.com/mongodb/mongo-php-library/pull/1439